lib/omniauth/authorization_failure_endpoint.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module OmniAuth
+  # rubocop:disable AbcSize
+  class AuthorizationFailureEndpoint < OmniAuth::FailureEndpoint
```

Comment:
Here we override the default OmniAuth failure endpoint in order to pass through the failing request parameters to our exception handler, as the default behavior is to simply swallow the request and redirect with no other context, which lead to a workaround of passing a fully qualified `error_url` in the `message` parameter which breaks when the url contains a querystring